codegen/templates/endpoint_test.tmpl

```diff
@@ -32,7 +32,7 @@ func Test{{.HandlerID | Title}}{{.TestName | Title}}OKResponse(t *testing.T) {
 	gateway, err := testGateway.CreateGateway(t, nil, &testGateway.Options{
 		KnownHTTPBackends: []string{"{{$clientName | camel}}"},
 		TestBinary: filepath.Join(
-			getDirName(), "..", "..", "main.go",
+			getDirName(), "..", "..", "services", "example-gateway", "main.go",
```

Comment:
I'd actually argue that we need to not hard couple to a binary in the endpoint tests. If you're generating tests that target a specific binary, they should be generated as part of the tests for that binary, not for a general purpose unit.